defaults/config.js

```diff
@@ -139,6 +139,29 @@ module.exports = {
 	// This value is set to `2048` kilobytes by default.
 	prefetchMaxImageSize: 2048,
 
+	// ### `upload`
```

Comment:
@astorije: if you could review this file that'd be great

Reply:
Nitpick: Do we want to call this `fileUpload`? `upload` alone could be confusing, especially as we add new features in the future. 🤷‍♂️

Reply:
So, an alternative to:

```
fileUpload: true,
fileUploadMaxSize: 10240,
```

is to nest it like we do in a few other places:

```
fileUpload: {
  enabled: true,
  maxFileSize: 10240,
}
```

Thoughts?